src/main/java/ua/softserveinc/tc/controller/util/ViewEventController.java

```diff
@@ -127,8 +121,17 @@ public void getEventForDelete(@RequestBody EventDto eventDto) {
 
     @RequestMapping(value = "getrecurrentevents", method = RequestMethod.POST, produces = "application/json")
     @ResponseBody
-    public String gerRecurrent(@RequestBody RecurrentEventDto recurrentEventDto) {
-        return new Gson().toJson(calendarService.createRecurrentEvents(recurrentEventDto));
+    public String gerRecurrent(@RequestBody RecurrentEventDto recurrentEventDto, BindingResult bindingResult) {
```

Comment:
i've added one more method for getting monthly event, 
is it possible to valdate it too?